### PR TITLE
Alt attributes for drop-shadow images on the CSS filters page.

### PR DESCRIPTION
**Pull request by goshdarnheck**
#### Summary
As mentioned in https://github.com/mdn/content/issues/19334 there are many missing or incomplete alt attributes for images. This PR adds some descriptive alt text for 8 images on the CSS filter property page.

#### Motivation
Accessibility! For anyone using a screen reader, or if the images don't load etc.

#### Supporting details
List of missing alt attributes: https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit?usp=sharing

#### Related issues
https://github.com/mdn/content/issues/19334

#### Metadata
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by goshdarnheck** [comment]
Tested w/ a screen reader.

File: files/en-us/web/css/filter/index.md [hunk 1]
```diff
@@ -405,11 +405,12 @@ filter: drop-shadow(16px 16px 10px black)
   </thead>
   <tbody>
     <tr>
-      <td><img id="img1" class="internal default" src="test_form_4.jpeg" style="width: 100%;" /></td>
-      <td><img id="img2" class="internal default" src="test_form_4.jpeg" style="width: 100%;" /></td>
+      <td><img alt="A rectangular image of a pencil marking a piece of paper. Most of the image is the light gray color of the paper." id="img1" class="internal default" src="test_form_4.jpeg" style="width: 100%;" /></td>
```
Comment (estelle) [inline files/en-us/web/css/filter/index.md:408]:
```suggestion
      <td><img alt="A rectangular image of a pencil marking a multiple choice Optical Mark Recognition, or OMR, form." id="img1" class="internal default" src="test_form_4.jpeg" style="width: 100%;" /></td>
```

File: files/en-us/web/css/filter/index.md [hunk 2]
```diff
@@ -405,11 +405,12 @@ filter: drop-shadow(16px 16px 10px black)
...
+      <td><img alt="A black drop shadow is applied to the image, which is visible below the image and to the right. The offset and blur-radius values produce a soft, rounded shadow." id="img2" class="internal default" src="test_form_4.jpeg" style="width: 100%;" /></td>
```
Comment (estelle) [inline files/en-us/web/css/filter/index.md:409]:
```suggestion
      <td><img alt="The OMR photo with a black drop shadow visible below the image and to the right. The offset and blur-radius values produce a soft, rounded shadow." id="img2" class="internal default" src="test_form_4.jpeg" style="width: 100%;" /></td>
```

File: files/en-us/web/css/filter/index.md [hunk 3]
```diff
@@ -405,11 +405,12 @@ filter: drop-shadow(16px 16px 10px black)
...
       <td>
         <div class="svg-container">
-          <svg xmlns="http://www.w3.org/2000/svg" id="img3" overflow="visible" viewBox="0 0 213 161" color-interpolation-filters="sRGB">
+          <svg aria-labelledby="svg-drop-shadow-title" role="img" xmlns="http://www.w3.org/2000/svg" id="img3" overflow="visible" viewBox="0 0 213 161" color-interpolation-filters="sRGB">
```
Comment (estelle) [inline files/en-us/web/css/filter/index.md:412]:
```suggestion
          <svg aria-labelledby="svg-drop-shadow-title svg-drop-shadow-desc" role="img" xmlns="http://www.w3.org/2000/svg" id="img3" overflow="visible" viewBox="0 0 213 161" color-interpolation-filters="sRGB">
```

File: files/en-us/web/css/filter/index.md [hunk 4]
```diff
@@ -405,11 +405,12 @@ filter: drop-shadow(16px 16px 10px black)
...
+            <title id="svg-drop-shadow-title">The same image as the previous two examples, but embedded within an SVG element. A filter element with a gaussian blur and offset is applied. The offset and blur values (set with the stdDeviation attribute on the feGaussianBlur element) produce a shadow that has less spread and is more rectangular than the previous CSS example. The shadow is visible below and to the right of the image, but the spread of the shadow is much less. The result is a shadow that is more closer to the rectangular shape of the original image.</title>
```
Comment (estelle) [inline files/en-us/web/css/filter/index.md:413]:
```suggestion
            <title id="svg-drop-shadow-title">Rectangular OMR photo</title>
            <desc id="svg-drop-shadow-desc">The same image as the previous two examples, but embedded within an SVG element. A filter element with a gaussian blur and offset is applied. The offset and blur values (set with the stdDeviation attribute on the feGaussianBlur element) produce a shadow that has less spread and is more rectangular than the CSS example. The shadow is visible below and to the right of the image, but with a smaller spread. The result is a shadow that is closer to the rectangular shape of the original image.</desc>
```

File: files/en-us/web/css/filter/index.md [hunk 5]
```diff
@@ -422,14 +423,15 @@ filter: drop-shadow(16px 16px 10px black)
           </svg>
         </div>
       </td>
-      <td><img id="img4" class="internal default" src="test_form_4_s.jpg" style="width: 100%;" /></td>
+      <td><img alt="A screenshot of the previous example image with drop shadow applied to show the effect in browsers that do not support the CSS or SVG filters." id="img4" class="internal default" src="test_form_4_s.jpg" style="width: 100%;" /></td>
```
Comment (estelle) [inline files/en-us/web/css/filter/index.md:426]:
```suggestion
      <td><img alt="A screenshot of the OMR image with drop shadow applied to show the effect for browsers that do not support the CSS or SVG filters." id="img4" class="internal default" src="test_form_4_s.jpg" style="width: 100%;" /></td>
```

File: files/en-us/web/css/filter/index.md [hunk 6]
```diff
@@ -422,14 +423,15 @@ filter: drop-shadow(16px 16px 10px black)
...
-      <td><img alt="test_form_4 distorted border - Live example" id="img12" class="internal default" src="test_form_4_irregular-shape_opacity-gradient.png" style="width: 100%;" /></td>
+      <td><img alt="An edited version of the original image that uses transparency to achieve an irregular shape at the bottom and add a small hole. The image also has a gradient transparency that starts on the right edge and fades towards the other side." id="img11" class="internal default" src="test_form_4_irregular-shape_opacity-gradient.png" style="width: 100%;" /></td>
+      <td><img alt="A black drop shadow is applied to the image, which is visible below the image and to the right. The shadow follows the curves of the irregular shaped bottom, and is visible through the hole as well. The shadow color has a slight transparency, so it displays as a dark gray over the white background of the page. Because of the transparent gradient across the image, the drop shadow is visible below most of the image, which makes it appear more gray than the original example." id="img12" class="internal default" src="test_form_4_irregular-shape_opacity-gradient.png" style="width: 100%;" /></td>
```
Comment (estelle) [inline files/en-us/web/css/filter/index.md:430]:
```suggestion
      <td><img alt="Image with black drop shadow applied which is visible below and to the right of the image. The shadow follows the curves of the irregularly shaped bottom and is also visible through the holes The shadow color is not fully opaque, so it displays as a dark gray over the white background of the page and is visible below most of the image, which makes it appear more gray than the original example." id="img12" class="internal default" src="test_form_4_irregular-shape_opacity-gradient.png" style="width: 100%;" /></td>
```

File: files/en-us/web/css/filter/index.md [hunk 7]
```diff
@@ -422,14 +423,15 @@ filter: drop-shadow(16px 16px 10px black)
...
       <td>
         <div class="svg-container">
-          <svg xmlns="http://www.w3.org/2000/svg" id="img13" overflow="visible" viewBox="0 0 213 161" color-interpolation-filters="sRGB">
+          <svg aria-labelledby="svg-irregular-drop-shadow-title" role="img" xmlns="http://www.w3.org/2000/svg" id="img13" overflow="visible" viewBox="0 0 213 161" color-interpolation-filters="sRGB">
```
Comment (estelle) [inline files/en-us/web/css/filter/index.md:433]:
```suggestion
          <svg aria-labelledby="svg-irregular-drop-shadow-title svg-irregular-drop-shadow-desc" role="img" xmlns="http://www.w3.org/2000/svg" id="img13" overflow="visible" viewBox="0 0 213 161" color-interpolation-filters="sRGB">
```

File: files/en-us/web/css/filter/index.md [hunk 8]
```diff
@@ -422,14 +423,15 @@ filter: drop-shadow(16px 16px 10px black)
...
+            <title id="svg-irregular-drop-shadow-title">The same image as the previous two examples, but embedded within an SVG element. A drop shadow has been added with a filter element using a feGaussianBlur and an feOffset element. The shadow follows the irregular shape of the image and is visible under the transparent gradient of the image.</title>
```
Comment (estelle) [inline files/en-us/web/css/filter/index.md:434]:
```suggestion
            <title id="svg-irregular-drop-shadow-title">OMR photo with irregular bottom edge</title>
            <desc  id="svg-irregular-drop-shadow-desc">The same image as the previous two examples, but embedded within an SVG element. A drop shadow has been added with a filter element using a feGaussianBlur and an feOffset element. The shadow follows the irregular shape of the image and is visible under the transparent gradient of the image.</desc>
```

**Review by estelle — COMMENTED** [review]
I love that you added aria-labelledby AND role img. Just made a few suggestions.

**Comment by goshdarnheck** [comment]
@estelle thanks so much for the great suggestions, I've merged them all in!

**Review by estelle — APPROVED** [review]
Thanks!

**Pull request merged**